src/lib/components/Frame.svelte

```diff
@@ -20,13 +20,21 @@
 
 	const selected = useSelected()
 	const events = useObjectEvents(() => uuid)
+
+	const colorGroup = getColorGroup(rest.name)
```

Comment:
```suggestion
	const colorGroup = $derived(getColorGroup(rest.name))
```
since name can change